aws_sagemaker_studio/frameworks/pytorch_cnn_cifar10/pytorch_cnn_cifar10.ipynb

```diff
@@ -4360,7 +604,6 @@
   }
  ],
  "metadata": {
-  "instance_type": "ml.t3.medium",
```

Comment:
is this intentional?

Reply:
Yes, it is - we do have enough quota for ml.t3.medium now, but the quota is only 400 compared to ml.m5.xlarge's quota of 2000 instances. So when I can, I try to switch it to the default.

Also, I never tested it but I wonder if ml.m5.xlarge runs faster - it will cut down on CI testing time.